components/Collections/examples/CollectionsInkExample.m

```diff
@@ -73,7 +73,7 @@ - (UIColor *)collectionView:(UICollectionView *)collectionView
   if (indexPath.item == 1) {
     return [MDCPalette.lightBluePalette.tint500 colorWithAlphaComponent:0.2f];
   } else if (indexPath.item == 2) {
-    return [UIColor colorWithRed:1.0 green:0.0 blue:0.0 alpha:0.2];
+    return [UIColor colorWithRed:1.0 green:0.0 blue:0.0 alpha:0.2f];
```

Comment:
Are the 1.0 and 0.0 values here ok?

Reply:
Done.
very odd. the compiler didn't complain about these.